helm-charts/infisical/README.md

```diff
@@ -91,6 +92,7 @@ kubectl get secrets -n <namespace> <secret-name> \
 | `backend.image.repository`                             | Backend image repository                                                                                                                                                                                                            | `infisical/backend`         |
 | `backend.image.tag`                                    | Backend image tag                                                                                                                                                                                                                   | `latest`                    |
 | `backend.image.pullPolicy`                             | Backend image pullPolicy                                                                                                                                                                                                            | `IfNotPresent`              |
+| `backend.resources`                                    | Backend container resource spesification, check the offical [kubernetes documentations](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/)                                                             | ``                          |
```

Comment:
How did you generate these docs? Please follow these steps to generate docs 

https://github.com/Infisical/infisical/blob/e47912edd79bd379f622a5b36af16bddc620d784/helm-charts/README.md?plain=1#L33-L39